packages/CopyInput/src/CopyInput.styles.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+
+export const CopyInput = styled.div`
+  display: flex;
+  [data-pka-anchor="heading"] {
+    margin-bottom: 0;
+  }
+
+  input.form-input__input {
```

Comment:
That `className` is an artifact left over from the migration from `acl-ui`.  We will likely remove it at some point.  You could use a `[data-pka-anchor="input"]` selector, but actually I think just `input` is fine too.

Reply:
I actually needed 2 classnames here or else the border-radius would get overridden by `Input` component styling. Maybe I could use `input[data-pka-anchor="input"]`?